### PR TITLE
Clean up language

### PR DESCRIPTION
**Pull request by gerrrg**
* Update documentation to be in line with language and terminology guidelines
* Fix numerous typos and poor phrasings throughout

### CODE REVIEW

File: docs/concepts/pools/boosted.md [hunk 1]
```diff
@@ -49,11 +48,12 @@ If this were instead to be built as a Boosted Pool, the idle assets can become y
 ## Case Study: bb-a-USD
 ![](/images/bb-a-USD.png)
 
-[Balancer Boosted Aave USD](https://app.balancer.fi/#/ethereum/pool/0xa13a9247ea42d743238089903570127dda72fe4400000000000000000000035d) (symbol: `bb-a-USD`) is a Composable Stable Pool that facilitates trades between three US Dollar stablecoins (`USDC`, `USDT`, and `DAI`) while sending idle liquidity to Aave. The underlying Linear Pools are:
+[Balancer Boosted Aave USD](https://app.balancer.fi/#/ethereum/pool/0xa13a9247ea42d743238089903570127dda72fe4400000000000000000000035d) (symbol: `bb-a-USD`) is a Composable Stable Pool that facilitates swaps between three US Dollar stablecoins (`USDC`, `USDT`, and `DAI`) while sending idle liquidity to Aave. The underlying Linear Pools are:
 * `bb-a-USDC` (consisting of `USDC` and wrapped `aUSDC`)
 * `bb-a-USDT` (consisting of `USDT` and wrapped `aUSDT`)
 * `bb-a-DAI` (consisting of `DAI` and wrapped `aDAI`)
 
+
```
Comment (gerrrg) [inline docs/concepts/pools/boosted.md]:
```suggestion
```

File: docs/concepts/pools/protocol.md [hunk 2]
```diff
@@ -1,38 +1,37 @@
 # Protocol Pools
 
-Unlike any of the other pool categories where the pool can be thought of as an indvidual entity, Protocol pools represent entire defi protocols built on top of Balancer infrastructure. Balancer provides the infrastructure for custom AMM’s by separating the pool logic from the accounting logic. Any AMM logic can be implemented as a custom pool on top of the Balancer vault and as a result, the design space of Balancer pools is infinite. In simpler terms: **programmable liquidity**.
+Unlike any of the other pool categories where the pool can be thought of as an indvidual entity, Protocol pools represent entire DeFi protocols built on top of Balancer infrastructure. Balancer provides the infrastructure for custom AMMs by separating the pool logic from the accounting logic.  Any AMM logic can be implemented as a custom pool on top of the Balancer Vault and as a result, the design space of Balancer pools is infinite. In simpler terms: **programmable liquidity**.
```
Comment (gerrrg) [inline docs/concepts/pools/protocol.md]:
```suggestion
Unlike any of the other pool categories where the pool can be thought of as an indvidual entity, Protocol pools represent entire DeFi protocols built on top of Balancer infrastructure. Balancer provides the infrastructure for custom AMMs by separating the pool logic from the accounting logic. Any AMM logic can be implemented as a custom pool on top of the Balancer Vault and as a result, the design space of Balancer pools is infinite. In simpler terms: **programmable liquidity**.
```

File: docs/reference/swaps/flash-swaps.md [hunk 3]
```diff
@@ -10,8 +10,7 @@ order: 3
 To understand Flash Swaps, make sure you understand [Batch Swaps](batch-swaps.md) first.
 :::
 
-Anyone who identifies a price discrepancy in two Balancer Pools can execute a **Flash Swap**. \***\* An arbitrageur who makes a flash swap **does not need to hold any** of the input **tokens\*\* that one would normally need to make a trade. Instead, the trader identifies the imbalance, tells the Vault to make the swap, and is rewarded with the profit.
-
+Anyone who identifies a price discrepancy in two Balancer Pools can execute a **Flash Swap**. \***\* An arbitrageur who makes a flash swap **does not need to hold any** of the input **tokens\*\* that one would normally need to make a swap. Instead, the arbitrageur identifies the imbalance, tells the Vault to make the swap, and is rewarded with the profit.
```
Comment (gerrrg) [inline docs/reference/swaps/flash-swaps.md]:
```suggestion
Anyone who identifies a price discrepancy in two Balancer Pools can execute a **Flash Swap**. An arbitrageur who makes a flash swap **does not need to hold any** of the input `tokens` that one would normally need to make a swap. Instead, the arbitrageur identifies the imbalance, tells the Vault to make the swap, and is rewarded with the profit.
```

**Review by mikemcdonald — APPROVED** [review]
